### PR TITLE
showcase: add additional AI projects and ESLint (format showcase.json properly again)

### PR DESCRIPTION
**Pull request by jennifertrin**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: showcase.json [hunk 1]
```diff
@@ -4099,6 +5184,48 @@
...
+    "name": "zCloak Network",
+    "description": "Defend trust and privacy in the age of AI by offering rapid, cost-effective, and secure zero-knowledge proof verification services for blockchains.",
+    "logo": "/img/showcase/zCloak_logo.svg",
+    "display": "Normal",
+    "oneLiner": "DID and ZKP Infrastructure for Web3",
+    "tags": [
+      "AI",
+      "Tools / Infrastructure"
+    ],
+    "usesInternetIdentity": true,
```
Comment (marc0olo) [inline showcase.json:5227]: which product uses internet identity? I am struggling struggling to find a login with II

File: showcase.json [hunk 2]
```diff
@@ -4099,6 +5184,48 @@
...
+    "name": "Blueband",
+    "description": "A vector database built based on Vectra, a local vector database for Node.js.",
+    "logo": "/img/showcase/blueband_logo.png",
+    "display": "Normal",
+    "oneLiner": "A vector database built based on Vectra, a local vector database for Node.js",
+    "tags": [
+      "AI"
+    ],
+    "usesInternetIdentity": false,
+    "github": "https://github.com/highfeast/ic-use-blueband-db"
```
Comment (marc0olo) [inline showcase.json:5200]: how is that project related to IC? 🤔

**Review by marc0olo — COMMENTED** [review]
@jennifertrin provided a few comments. if possible we should check/enforce the json formatting so that we do not have so many changes again

File: showcase.json [hunk 3]
```diff
@@ -4,55 +4,88 @@
     "name": "Helix Markets",
     "oneLiner": "Bringing true ownership and full transparency to crypto trading",
     "website": "https://www.helixmarkets.io/",
-    "tags": ["Ethereum", "Bitcoin", "DeFi", "Chain Fusion"],
+    "tags": [
```
[the remark this answers is not in the document]
Comment (jennifertrin) [reply showcase.json:7]: Someone must have updated it and doesn't use ESLint. I personally like ESLint and I know so does the eng content team. I'll have to circle up with the eng content team if they really want to enforce it.

File: showcase.json (hunk 1 above)
Comment (jennifertrin) [reply showcase.json]: Ah yes, I meant to say false here. I updated it.

**Review by marc0olo — APPROVED** [review]
lgtm now 👍 

I think we should provide more context to projects like zCloak if possible to get more information about the relation to ICP

**Review by jessiemongeon1 — APPROVED** [review]
